FTBFS visit_all_subdirs fallthrough

**Describe the bug**
```
tumbleweed-pkg/t/s/src (main)$ make CC=gcc
gcc -std=c11 -c -Wall -Wextra -Werror -O3 -flto -fno-asynchronous-unwind-tables -fno-unwind-tables -I/usr/include/json-c  -I/usr/include/python3.11 -I/usr/include/python3.11 -DUSE_PYTHON -MT alsa.o -MMD -MP -MF ./alsa.Td -o alsa.o alsa.c
mv -f ./alsa.Td ./alsa.d && touch alsa.o
gcc -std=c11 -c -Wall -Wextra -Werror -O3 -flto -fno-asynchronous-unwind-tables -fno-unwind-tables -I/usr/include/json-c  -I/usr/include/python3.11 -I/usr/include/python3.11 -DUSE_PYTHON -MT poller.o -MMD -MP -MF ./poller.Td -o poller.o poller.c
mv -f ./poller.Td ./poller.d && touch poller.o
gcc -std=c11 -c -Wall -Wextra -Werror -O3 -flto -fno-asynchronous-unwind-tables -fno-unwind-tables -I/usr/include/json-c  -I/usr/include/python3.11 -I/usr/include/python3.11 -DUSE_PYTHON -MT process_configuration.o -MMD -MP -MF ./process_configuration.Td -o process_configuration.o process_configuration.c
mv -f ./process_configuration.Td ./process_configuration.d && touch process_configuration.o
gcc -std=c11 -c -Wall -Wextra -Werror -O3 -flto -fno-asynchronous-unwind-tables -fno-unwind-tables -I/usr/include/json-c  -I/usr/include/python3.11 -I/usr/include/python3.11 -DUSE_PYTHON -MT utility.o -MMD -MP -MF ./utility.Td -o utility.o utility.c
utility.c: In function ‘visit_all_subdirs’:
utility.c:328:20: error: this statement may fall through [-Werror=implicit-fallthrough=]
  328 |                 if (!isdir(path, path_fd, ent->d_name))
      |                    ^
utility.c:331:13: note: here
  331 |             case DT_DIR:
      |             ^~~~
cc1: all warnings being treated as errors
make: *** [Makefile:76: utility.o] Error 1
tumbleweed-pkg/t/s/src (main)$
```

**Expected behaviour**
Clear build

**Actual behaviour**
Fails to build

**Desktop (please complete the following information):**
 - The Linux flavour you use: openSUSE/Tumbleweed
 - `swaystatus` version: master version from git
 - `swaystatus` configuration none

--- a/src/utility.c
+++ b/src/utility.c
@@ -327,6 +327,7 @@ void visit_all_subdirs(const char *path, subdir_visiter visiter, ...)
                 // Check if it is a dir after resolution
                 if (!isdir(path, path_fd, ent->d_name))
                     break;
+                [[fallthrough]];
 
             case DT_DIR:
                 if (strcmp(ent->d_name, ".") != 0 && strcmp(ent->d_name, "..") != 0)

